### DISCUSSION TITLE
Question about ReBarState for a testbench.

### DISCUSSION THREAD

**Discussion by JamSWACD** [xCuri0/ReBarUEFI#159] [General]
I have used ReBarUEFI to enable Resizable Bar on my X99 testbench. If I wanted to swap from a UEFI-capable GPU to one that requires CSM enabled, would I just need to disable the BAR with ReBARState.exe before swapping GPU? Or would there still be problems? Thanks!

**Comment by xCuri0** [comment]
@JamSWACD probably not because the GPU that requires CSM likely won't have rebar
